utils/graph.c

```diff
@@ -10,6 +10,19 @@ static void *cb_arg;
 
 static struct rb_root task_graph_root = RB_ROOT;
 
+static void init_time_stat(struct graph_time_stat *ts)
+{
+	ts->min = -1ULL;
```

Comment:
Please init other fields too.

Reply:
@namhyung @honggyukim

```
static void init_time_stat(struct report_time_stat *ts)
{
	ts->min = -1ULL;
}
``` 
ref: https://github.com/namhyung/uftrace/blob/master/utils/report.c#L11-L14

`init_time_state` function in `utils/report.c` file it didn't init too except min.
does it necessary?

Reply:
Then you can use it when building partial graphs.